mandatory/3-function-output.js

```diff
@@ -1,16 +1,19 @@
 // Add comments to explain what this function does. You're meant to use Google!
+/* The math.random() function return a random number between 0 and 1 but when we multiply by 100 it return a random number between 0 and 100;*/
+
 function getRandomNumber() {
   return Math.random() * 10;
 }
-
 // Add comments to explain what this function does. You're meant to use Google!
+/* The concat() function combind word2 to word1 or we can say it push word2 to word1 . */
 function combine2Words(word1, word2) {
   return word1.concat(word2);
 }
 
 function concatenate(firstWord, secondWord, thirdWord) {
   // Write the body of this function to concatenate three words together.
   // Look at the test case below to understand what this function is expected to return.
+  return `${firstWord} ${secondWord} ${thirdWord}`;
```

Comment:
This works! Another way to do it could be:
var firstAndSecondWord = firstWord.concat(secondWord);
var firstAndSecondAndThirdWord = firstAndSecondWord.concat(thirdWord);


or you could chain them together:
return firstWord.concat(secondWord).concat(thirdWord);


or another way would be to use the function above:
var firstAndSecondWord = combine2Words(firstWord, secondWord);
var firstAndSecondAndThirdWord = combine2Words(firstAndSecondWord, thirdWord);

or chain this:
return combine2Words(combine2Words(firstWord, secondWord), thirdWord);


There are usually loads of ways to do the same thing. As long as it works and is readable then any will do!